#! /bin/sh
patch -p1 -l -f $* < $0
exit $?

From 32df15740e9b2581526a9f77b2906c8033b39042 Mon Sep 17 00:00:00 2001
From: Andre Heinecke <aheinecke@gnupg.org>
Date: Mon, 8 Jul 2024 16:20:30 +0200
Subject: [PATCH] Fix function signature of page_info

Remove wrong stdcall macro from page_info.
---
 src/lib/util/kmemoryinfo.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/lib/util/kmemoryinfo.cpp b/src/lib/util/kmemoryinfo.cpp
index 1787c26a..30efa2cd 100644
--- a/src/lib/util/kmemoryinfo.cpp
+++ b/src/lib/util/kmemoryinfo.cpp
@@ -150,7 +150,7 @@ struct SwapInfo {
     quint64 freePageFilePages = 0;
 };

-BOOL __stdcall pageInfo(LPVOID pContext, PENUM_PAGE_FILE_INFORMATION pPageFileInfo, LPCWSTR lpFilename)
+BOOL pageInfo(LPVOID pContext, PENUM_PAGE_FILE_INFORMATION pPageFileInfo, LPCWSTR lpFilename)
 {
     Q_UNUSED(lpFilename)
     if (auto sw = static_cast<SwapInfo *>(pContext)) {
--
2.45.2
